libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigationProvider.kt

```diff
@@ -45,6 +55,9 @@ object MapboxNavigationProvider {
      * Destroy MapboxNavigation when your process/activity exits.
      */
     @JvmStatic
+    @Deprecated(
+        message = "MapboxNavigationApp will determine when to destroy MapboxNavigation instances"
+    )
```

Comment:
If possible, let's provide `ReplaceWith` annotation to allow IDEs to automatically apply the replacements. ([here an example](
https://github.com/mapbox/mapbox-navigation-android/blob/47e17155e6766a84b6de012d8e6874d65378ffcd/libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/camera/data/MapboxNavigationViewportDataSource.kt#L395))

Reply:
There is no `ReplaceWith` here, there is no use for this function in `MapboxNavigationApp`